keps/sig-network/2595-expanded-dns-config/README.md

```diff
@@ -138,7 +146,8 @@ DNS configuration
 #### Alpha -> Beta Graduation
 
 - Address feedback from alpha
-- Sufficient testing
+- Add e2e tests
+- Wait for major container runtimes to support this feature
```

Comment:
this is tricky, since we can't support this feature if users can still use  runtimes versions that doesn't support expanded DNS. Per example,  containerd support it in 1.5.5, but 1.5.1 is still supported
We should have something like
"all major container runtimes supported versions allow expanded DNS configurations"